cypress/e2e/PageObjects/CPPContactUsPO.cy.js

```diff
@@ -0,0 +1,142 @@
+/// <reference types="cypress" />
+import ContactUsCPPdata from '../../fixtures/ContactUsCPPdata.json'
+
+function CPPContactUsHeader() {
+  return cy.get('[data-cy ="eIContactUsContent"]>h1')
+}
+
+function ValidateCPPContactUsUrl() {
+  cy.url().should('contains', '/contact-us/contact-canada-pension-plan')
+}
+
+function ValidateCPPContactUsHeaderEN() {
+  CPPContactUsHeader()
+    .should('be.visible')
+    .and('have.text', 'Contact Canada Pension Plan')
+}
+
+function ValidateCPPContactUsUrlFR() {
+  cy.url().should('contains', '/fr/contact-us/contact-canada-pension-plan')
```

Comment:
done

> Looks great! We'll just have to swap out the french URLs with the new updated URLs once you merge in dev so this would be /fr/contactez-nous/communiquer-regime-pensions-canada/

changed! Thank you